airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -170,3 +184,51 @@ def download_line_items(self, request_body: Dict[str, Any]) -> List[Any]:
             .execute(num_retries=self.num_retries)
         )
         return response["lineItems"]
+
+    def create_sdf_download_operation(self, body_request: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Creates an SDF Download Task and Returns an Operation.
+        :param body_request: Body request.
```

Comment:
```suggestion

        :param body_request: Body request.
```

Reply:
The documentation will not be built without it.

Reply:
Hm... Ok, thanks. 
I did not have issues when I built documentation locally. I will fix it.